mainsite/catalog/views.py

```diff
@@ -13,6 +14,15 @@
 from datetime import datetime, timedelta
 import pytz
 
+# This helper function checks if a user is currently banned / timed out
+def isCurrentlyBanned(username):
```

Comment:
Bit of a nitpick but the naming now doesn't make sense given the return value. Maybe do find+replace for `isCurrentlyBanned` -> `isUserNotBanned` or similar?